mobile/library/common/common/system_helper.h

```diff
@@ -26,6 +28,16 @@ class SystemHelper {
    */
   virtual bool isCleartextPermitted(absl::string_view hostname) PURE;
 
+  /**
+   * Invokes platform APIs to validate certificates.
+   */
+  virtual envoy_cert_validation_result
+  validateCertificateChain(const std::vector<std::string>& certs, absl::string_view hostname) PURE;
```

Comment:
nit: add a newline below to be consistent w/ the rest of the file

Reply:
Done!